pages/_meta.json

```diff
@@ -9,6 +9,7 @@
   "governance": "Governance",
   "security": "Security",
   "terms_and_policies": "Terms and Policies",
+  "deposits_and_withdrawals": "Deposits and Withdrawals",
```

Comment:
can we move this to be under trading? 

we should keep terms and policies to be last and should group activities that users can do together

Reply:
Done